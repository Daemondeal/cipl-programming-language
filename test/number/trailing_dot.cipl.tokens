Token          Lexeme         Line 
IDENTIFIER     println        1    
LEFT_PAREN     (              1    
NUMBER         123            1     123
DOT            .              1    
RIGHT_PAREN    )              1    
LINE_END       \n             1    
EOF                           1    
