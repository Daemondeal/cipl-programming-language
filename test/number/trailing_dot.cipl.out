::stdout::

::stderr::
Error on line 1 at `\)`: Expect property name after '.'
